src/main/java/de/unistuttgart/overworldbackend/service/DungeonService.java

```diff
@@ -0,0 +1,91 @@
+package de.unistuttgart.overworldbackend.service;
+
+import de.unistuttgart.overworldbackend.data.Dungeon;
+import de.unistuttgart.overworldbackend.data.DungeonDTO;
+import de.unistuttgart.overworldbackend.data.mapper.DungeonMapper;
+import de.unistuttgart.overworldbackend.repositories.DungeonRepository;
+import java.util.Set;
+import java.util.UUID;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+import org.springframework.web.server.ResponseStatusException;
+
+@Service
+public class DungeonService {
+
+  @Autowired
+  private DungeonRepository dungeonRepository;
+
+  @Autowired
+  private DungeonMapper dungeonMapper;
+
+  /**
+   * Get a dungeon of a lecture and a world
+   *
+   * @throws ResponseStatusException if dungeon with its id could not be found in the lecture
```

Comment:
```suggestion
   * @throws ResponseStatusException(404) if no matching dungeon could not be found
```

Reply:
`@throws ResponseStatusException(404)` is this the right syntax for java doc? Do you have a source for me?

Reply:
It's probably not the right syntax.
I have no idea how to display it otherwise but it is important to note which return value is specifically thrown.
Otherwise, you simply know that a custom status is thrown, but you don't know which.
What about
```suggestion
   * @throws ResponseStatusException (404) if no matching dungeon could not be found
```
as an alternative?

Reply:
looks ok. What if one function throws a 404 and a 400. Should we create two `@throws` or just mention the two different http codes in the one `@throws`

Reply:
one `@throws`, different http codes.
Otherwise we'll probably confuse the `javadoc` tool

Reply:
added everywhere where we has `@throws ResponseStatusException`